panther_hardware_interfaces/src/panther_system.cpp

```diff
@@ -0,0 +1,519 @@
+// Copyright 2023 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <panther_hardware_interfaces/panther_system.hpp>
+
+#include <rclcpp/logging.hpp>
+
+#include <hardware_interface/types/hardware_interface_type_values.hpp>
+
+#include <panther_hardware_interfaces/utils.hpp>
+
+// todo: add user variable to the script that will trigger DOUT4 instead of safety stop
+
+namespace panther_hardware_interfaces
+{
+
+using namespace std::literals;
+
+void PantherSystem::CheckJointSize() const
+{
+  if (info_.joints.size() != kJointsSize) {
+    throw std::runtime_error(
+      "Wrong number of joints defined: "s + std::to_string(info_.joints.size()) + ", "s +
+      std::to_string(kJointsSize) + "expected."s);
```

Comment:
```suggestion
    throw std::runtime_error(
      "Wrong number of joints defined: " + std::to_string(info_.joints.size()) + ", " +
      std::to_string(kJointsSize) + "expected.");
```
Is this necessary? Only in this file you are doing it like this

Reply:
Apparently it is not, when I was writing it was necessary to have `string`, but now it compiles fine without it, so I will delete it.